tests/compile/test_fusion_attn.py

```diff
@@ -296,28 +314,49 @@ def forward(self, q: torch.Tensor, k: torch.Tensor, v: torch.Tensor):
                                      out_dtype=attn_output.dtype)
 
 
-@pytest.mark.parametrize("num_qo_heads, num_kv_heads", [(64, 8), (40, 8)])
+CUDA_MODELS = [("nvidia/Llama-4-Scout-17B-16E-Instruct-FP8",
+                TestAttentionFp8StaticQuantPatternModel),
+               ("nvidia/Llama-4-Scout-17B-16E-Instruct-FP4",
+                TestAttentionNvfp4QuantPatternModel)]
+ROCM_MODELS = [("amd/Llama-3.1-8B-Instruct-FP8-KV",
+                TestAttentionFp8StaticQuantPatternModel)]
+
+CUDA_HEADS = [(64, 8), (40, 8)]
+ROCM_HEADS = [(32, 8), (40, 8)]
```

Comment:
Why not just a single if statement:
```suggestion
if platform.is_cuda()
    MODELS = [("nvidia/Llama-4-Scout-17B-16E-Instruct-FP8",
               TestAttentionFp8StaticQuantPatternModel),
              ("nvidia/Llama-4-Scout-17B-16E-Instruct-FP4",
               TestAttentionNvfp4QuantPatternModel)]
    NUM_HEADS = [(64, 8), (40, 8)]
    # ...
else:
    MODELS = [("amd/Llama-3.1-8B-Instruct-FP8-KV",
               TestAttentionFp8StaticQuantPatternModel)]
    NUM_HEADS = [(32, 8), (40, 8)]
    # ...
```

Reply:
I considered this approach, but this implies that if it is not cuda, it's automatically rocm without clarification, which may not always remain true. I can change it if you want

Reply:
You could do `elif rocm: ... else: MODELS=[] ...`